backend/hardhat.config.ts

```diff
@@ -244,22 +274,50 @@ task('reclaimFunds')
     console.log(`Successfully reclaimed funds to ${args.payoutAddress}. Transaction hash: ${receipt!.hash}`);
   });
 
-// Add a new question.
+// Set gasless key-pair.
 task('setGaslessKeyPair')
   .addPositionalParam('address', 'contract address')
   .addPositionalParam('payerAddress', 'payer address')
   .addPositionalParam('payerSecret', 'payer secret key')
   .setAction(async (args, hre) => {
-    await hre.run('compile');
+    const quiz = await hre.ethers.getContractAt('Quiz', args.address);
+    const nonce = await hre.ethers.provider.getTransactionCount(args.payerAddress);
+    await setGaslessKeyPair(quiz, args.payerAddress, args.payerSecret, nonce);
+  });
 
-    let quiz = await hre.ethers.getContractAt('Quiz', args.address);
 
-    const nonce = await hre.ethers.provider.getTransactionCount(args.payerAddress);
-    const tx = await quiz.setGaslessKeyPair(args.payerAddress, args.payerSecret, nonce);
-    const receipt = await tx.wait();
-    console.log(`Successfully set gasless keypair to ${args.payerAddress}, secret ${args.payerSecret} and nonce ${nonce}. Transaction hash: ${receipt!.hash}`);
+// Deploy and setup Quiz contract.
+task('deployAndSetupQuiz')
+  .addOptionalParam('questionsfile', 'File containing questions in JSON format', 'test-questions.json')
```

Comment:
Use camel case here and in the rest of optional params. Hardhat will then automatically replace it to `--questions-file`. Don't forget to update the readme accordingly.

```suggestion
  .addOptionalParam('questionsFile', 'File containing questions in JSON format', 'test-questions.json')
```